.github/workflows/Pipeline.yml

```diff
@@ -12,6 +12,7 @@ jobs:
     uses: pyTooling/Actions/.github/workflows/Parameters.yml@r0
     with:
       name: pyEDAA.ToolSetup
+      system_list: 'ubuntu windows macos'  # disabled: "msys2"
```

Comment:
What's the reason for disabling testing on MSYS2?

Reply:
There is currently a slash vs. backslash problem.
It's already on again on `dev` branch. It will be fixed with next revision.